utils.py

```diff
@@ -100,37 +103,20 @@ def __getitem__(self, idx):
         return X, y
 
 
-def pong_state_processor(img, state_shape):
-    """Resize states to 64x64 with cropping suited for Pong.
+def state_processor(img, state_shape, crop_range):
+    """Resize states to `state_shape` with cropping of `crop_range`.
 
     Args:
         img (np.ndarray): Image to crop and resize.
         state_shape (tuple): Output shape.
+        state_shape (tuple): Range to crop.
```

Comment:
Popraw doca (crop_range, string, dopisz jakiś przykład, albo daj wartość domyślną).

Reply:
Ok

Reply:
Tutaj chyba pomyłka w copy-paste, jest dwa razy `state_shape`. I dodaj jeszcze coś w stylu "Range to crop in Python slicing syntax e.g. [30:183, 28:131, :]".

EDIT: Dopiero teraz zrozumiałem, że Grzegorz napisał to samo xD